pom.xml

```diff
@@ -46,21 +46,21 @@
         <camel.docs.branch>camel-${camel.major.minor}.x</camel.docs.branch><!-- The stable camel branch on which our Antora docs depends -->
         <camel.sb.docs.branch>camel-spring-boot-${camel.major.minor}.x</camel.sb.docs.branch><!-- The stable camel-spring-boot branch on which our Antora docs depends -->
         <cassandra-quarkus.version>1.2.0</cassandra-quarkus.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/com/datastax/oss/quarkus/cassandra-quarkus-bom/ -->
-        <debezium.version>2.5.0.Final</debezium.version> <!-- This should be in sync with quarkus-platform https://github.com/quarkusio/quarkus-platform-->
-        <optaplanner.version>9.37.0.Final</optaplanner.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/org/optaplanner/optaplanner-quarkus/ -->
-        <quarkiverse-amazonservices.version>2.10.1</quarkiverse-amazonservices.version><!-- This should be in sync with quarkus-platform https://repo1.maven.org/maven2/io/quarkiverse/amazonservices/quarkus-amazon-services-parent/ -->
+        <debezium.version>2.5.1.Final</debezium.version> <!-- This should be in sync with quarkus-platform https://github.com/quarkusio/quarkus-platform-->
```

Comment:
For Debezium, Optaplanner & AWS, we probably don't want to align ourselves to versions ahead of what is currently active in the platform.

Reply:
> How could it be checked what is the active version

You can look at the versions listed in the quarkus-platform pom.xml or in the individual generated member BOMs.

Tika has been broken for some time anyway https://github.com/apache/camel-quarkus/issues/5234.

Reply:
> it's looks we are ahead with quarkus-hazelcast-client.version

I think it's safe in this case since the Hazelcast member is disabled in the platform.

I guess they never bothered to upgrade when Quarkus 3.x was first released.

Reply:
ok, reverted tika, let's see how it goes.